src/Entities/ChatMember/Factory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace Longman\TelegramBot\Entities\ChatMember;
+
+
+use Longman\TelegramBot\Entities\Entity;
+use Longman\TelegramBot\Exception\TelegramException;
+
+class Factory extends \Longman\TelegramBot\Entities\Factory
+{
+    public function make(array $data, string $bot_username): Entity
+    {
+        if (! isset($data['status'])) {
```

Comment:
I think this can be removed.

If something weird is coming in (which is hugely unlikely), just fall back to returning the `ChatMemberNotImplemented` object.